ext/mysqli/mysqli.stub.php

```diff
@@ -850,9 +850,9 @@ public function get_server_info(): string {}
     public function get_warnings(): mysqli_warning|false {}
 
     /**
-     * @deprecated
      * @return bool|null
      * */
+    #[\Deprecated]
```

Comment:
Since 8.1.

Memo to doc team, this does not have the deprecated note at the top of the page but does have a changelog.

`The object-oriented style mysqli::init() method has been deprecated. Replace calls to parent::init() with parent::__construct(). `